src/test/java/com/kodilla/ecommercee/repository/GroupRepositoryTest.java

```diff
@@ -0,0 +1,41 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.Group;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.util.ArrayList;
+
+import static org.junit.Assert.*;
+
+@SpringBootTest
+@RunWith(SpringJUnit4ClassRunner.class)
+public class GroupRepositoryTest {
+    @Autowired
+    private GroupRepository groupRepository;
+
+    @Test
+    @DirtiesContext(methodMode = DirtiesContext.MethodMode.AFTER_METHOD)
+    public void crudTest() {

```

Comment:
tutaj powinniśmy to rozbić na 4 testy, ewentualnie zasilić wstępnie bazę jakąś sql-ką